ui/desktop/app/routes/scopes.js

```diff
@@ -6,14 +6,35 @@
 import Route from '@ember/routing/route';
 import { A } from '@ember/array';
 import { inject as service } from '@ember/service';
+import { set, get } from '@ember/object';
 
 export default class ScopesRoute extends Route {
   // =services
 
   @service store;
+  @service ipc;
 
   // =methods
 
+  async beforeModel() {
+    let isClientDaemonSupported = true;
```

Comment:
Should this by default be set to `false`? Since if in the try catch if it errors out now it will be defaulted to `true`? Is that what we want?

Reply:
Yeah that is a good point. I will update this to default to `false`.

Reply:
Updated.